src/map/next_js/resolveAxios.ts

```
@@ -5,8 +5,58 @@ import { Chunks } from "../../utility/interfaces.js";
+        case "ObjectExpression":
+            const props = node.properties.map((prop) => {
+                if (prop.type === "ObjectProperty") {
+                    const key =
+                        prop.key.type === "Identifier"
+                            ? `"${prop.key.name}"`
+                            : astNodeToJsonString(prop.key, code);
+                    const value = astNodeToJsonString(prop.value, code);
+                    return `${key}: ${value}`;
+                }
+                return '""'; // SpreadElement not handled
+            });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)